utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -161,6 +163,9 @@ struct WriteBatchIndexEntry {
   // SeekForPrev() will see all the keys with the same key.
   size_t offset;
   uint32_t column_family;  // column family of the entry.
+  // The following three fields are only maintained when the WBWI is created
+  // with overwrite_key = true.
+  uint32_t update_count;   // number of updates for this key up to this entry.
```

Comment:
Let's mention explicitly somewhere that this is a 1-based count, not 0-based

Reply:
Yes, updated comments.